GillianCore/engine/Abstraction/Verifier.ml

```diff
@@ -1045,8 +1047,9 @@ struct
     Printf.printf "%s\n" msg;
     L.normal (fun m -> m "%s" msg)
 
-  let verify_up_to_procs (prog : prog_t) :
+  let verify_up_to_procs ?(proc_name : string option) (prog : prog_t) :
       SAInterpreter.result_t SAInterpreter.cont_func =
+    proc_name |> ignore;
```

Comment:
?

Reply:
Resolved in c0f957eaf54d88a6a982d9dd569a5cc9a30f4d56